html/obsolete/requirements-for-implementations/frames/document-frameset-color.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<title>document frameset color</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+
+<script>
+test(function() {
+
+var fset = document.createElement("frameset");
+document.documentElement.appendChild(fset);
+
+document.fgColor = "blue";
+document.bgColor = "blue";
+document.linkColor = "blue";
+document.vlinkColor = "blue";
+document.alinkColor = "blue";
+
+assert_equals(document.fgColor,"");
+assert_equals(document.bgColor,"");
+assert_equals(document.linkColor,"");
+assert_equals(document.vlinkColor,"");
+assert_equals(document.alinkColor,"");
+
+document.documentElement.removeChild(fset);
+
+}
+//setting the document color attributes with a frameset
+);
+
+test(function() {
```

Comment:
I'm not sure it makes sense to separate the tests like this; at least since it's the same document, it is affected by what the previous test does. To keep it simple I would suggest doing everything in a single `test`...